react-native-libraries.json

```diff
@@ -10101,5 +10101,14 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/bidah/react-native-vercel-ai",
+    "npmPkg": "react-native-vercel-ai",
```

Comment:
Hello @bidah, thank you for adding new entry to the directory! 👍 

```suggestion
```

Just one nit before we ship, `npmPkg` is mostly dedicated for scoped packages, in case of you package it can be omitted.